### ISSUE TITLE
Exception Not Calling ,if user denied permission again If they click gallery ,I tried to navigate settings

### ISSUE THREAD

**Issue by shangarivivek** [Sh1d0w/multi_image_picker#220]
**Bug Report**

`multi_image_picker` version: 

Is it happening on Android: 

Is it happening on iOS: 

```
 <!-- run `flutter doctor` and paste the output here: -->
```

**Describe the bug**

**Comment by sunderls** [comment]
+1

I just ran into this problem also.

when I deny permission in settings, the exception handler is not called at all
```
try {
      var images = await MultiImagePicker.pickImages(
          maxImages: 6 - _images.length, enableCamera: true);
    } on Exception catch (e) {
     // not called at all
  } 
```

**Comment by Sh1d0w** [comment]
@sunderls Is it happening on iOS only? I've tested with Android and is working fine. It does not fire on iOS for some reason, I am going to release a hot fix, but can you confirm the problem on your side is only on iOS?

**Comment by Sh1d0w** [comment]
I've just released version 4.5.1 which fixes the problem on iOS. Please refer to the error handling docs for more info https://sh1d0w.github.io/multi_image_picker/#/errorhandling
